### PR TITLE
feat(prettier): add Hardline in empty block

### PR DESCRIPTION
**Pull request by Dunqing**

### CODE REVIEW

**Comment by Dunqing** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1369** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1369?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #1370** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1370?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #1371** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/1371?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/oxc-project/oxc/1371?utm_source=stack-comment).

File: crates/oxc_prettier/src/format/block.rs [hunk 1]
```diff
@@ -25,6 +25,20 @@ pub(super) fn print_block<'a>(
     Doc::Array(parts)
 }
 
+pub(super) fn print_statement_block<'a>(
```
Comment (Boshen) [inline crates/oxc_prettier/src/format/block.rs:28]: I'm trying to layout our code to match exactly like prettier. There is no `print_statement_block` in `src/language-js/print/block.js` 🤔

Comment (Dunqing) [reply crates/oxc_prettier/src/format/block.rs:28]: > I'm trying to layout our code to match exactly like prettier. There is no `print_statement_block` in `src/language-js/print/block.js` 🤔

Yes, I added this myself.

Comment (Dunqing) [reply crates/oxc_prettier/src/format/block.rs:28]: We can hold it until I refactor it

**Pull request closed without merging**